src/content/docs/serverless-function-monitoring/azure-function-monitoring/linux.mdx

```diff
@@ -0,0 +1,327 @@
+                #   initialization and registration
+                # 2. Moves the original function_app.py file into a directory
+                #    named "original". 
+                # 3. Adds the "original" directory to the .funcignore file
+                #   to prevent duplication in published files.
+                #
+                #####################################################################
+
+                mkdir original
+                mv function_app.py ./original/
+
+                echo -e 'import newrelic.agent\nimport os\n\nnewrelic.agent.initialize()\napp_name = os.environ.get("NEW_RELIC_APP_NAME", os.environ.get("WEBSITE_SITE_NAME", None))\nnewrelic.agent.register_application(app_name)\n' | cat - ./original/function_app.py > function_app.py
+
+                echo 'original' >> .funcignore
+              
+              ```
+
+            3. Save the `initialization.sh` file.
+            4. Provide the permissions to script file by running:
+            
+              ```bash
+              chmod 755 initialization.sh
+              ```
+            5. Run the script from the root directory of your project:
+            
+              ```bash
+              ./initialization.sh
+              ```
+            </Collapser>
+
+        </CollapserGroup>
+
+    4. Publish your Azure Function App using the Azure CLI or your preferred publishing method. You can use the following command to publish your Azure Function App:
+    
+        ```bash
+        func azure functionapp publish \ ${NAME_OF_YOUR_AZURE_FUNCTION_APP}
+        ```
+
+    </TabsPageItem>
+
+    <TabsPageItem id="net">
+
+    1. Add the latest version of the `NewRelic.Agent` NuGet package to your application project.
+    2. Use your preferred publishing mechanism to deploy your updated application to Azure. 
+    
+    
+    When you deploy your application, the .NET agent is installed in the `/home/site/wwwroot/newrelic` folder.
+    
+    </TabsPageItem>
+
+  </TabsPages>
+</Tabs>
+
+</Step>
+
+<Step>
+
+## Configure environment variables
+
+After you've published the Azure Function app, [configure environment variables](/docs/serverless-function-monitoring/azure-function-monitoring/env-variables-azure):
+
+  1. Navigate to your Azure Function in the Azure portal.
+  2. Under **Settings**, click **Environment variables**, and then click **Advanced edit**.
+  3. Based on your deployment runtime, paste the following values:
+
+<Callout variant="important">
+Ensure that you add a comma at the end of the last existing line and update your license key in the following configurations.
+</Callout>
+
+<Tabs>
+  <TabsBar>
+    
+    <TabsBarItem id="nodejs">Node.js</TabsBarItem>
+    <TabsBarItem id="py">Python</TabsBarItem>
+    <TabsBarItem id=".net">.NET</TabsBarItem>
+
+  </TabsBar>
+
+  <TabsPages>
+
+
+  <TabsPageItem id="nodejs">
+
+  ```json
+  {
+    Information required for Node.js agent
```

Comment:
"NODE_OPTIONS": "-r newrelic",
"NEW_RELIC_LICENSE_KEY": "licenseKey",
"NEW_RELIC_APP_NAME": "Name of Function App"